lib/xclarity_client/config_target.rb

```diff
@@ -0,0 +1,16 @@
+module XClarityClient
+  class ConfigTarget
+    include XClarityClient::Resource
+
+    BASE_URI = '/config/target'.freeze
+    LIST_NAME = 'items'.freeze
+
+    attr_accessor :access, :children, :description, :firmwareLevel, :id, :ipaddresses, :name, :type, :uuid
```

Comment:
I notice that _ConfigProfile_ has _identifier_, _items_, and _label_ attributes. Are they also needed here?
Also, there seems to be several fields in the API documentation that are not here. Not saying that they are needed, I just wanted it to bring it to your attention.

Reply:
Valid point. I missed these from my end. Have added them in the check-in done today